### ISSUE TITLE
Sprint 6 -- a11y

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#183]
- [x] Enhancement: From the section 508 review, the coordinator suggested in adding tab stop at the instructions so folks don't miss this before going to the form elements
+ [x] Remove extra space after protected class next button. Hopefully small, but the "next" button the protected class page seems to have a space included after it so it announces "next button space" on a screen reader when you use the up/down arrow keys. The Contact page button doesn't have this extra space.

**Comment by Jacklynn** [comment]
Labeling the tab stops as an enhancement that could roll over to the next sprint.

**Comment by Jacklynn** [comment]
I added the leftover Next button issue to this card

**Comment by alexsoble** [comment]
Tabbing changes implemented here: https://github.com/usdoj-crt/crt-portal/pull/178.

These improvements are now live on the dev site!

**Comment by alexsoble** [comment]
@Jacklynn, I can't reproduce the "Next" button issue you mention — it reads as "Next" for me on the protected class page in VoiceOver, not "Next space." 

![next-vo.png](https://images.zenhubusercontent.com/570e4e98bf93871ccd995e5b/b21b450e-6b45-46cb-8cbe-5435467efc91)

The code for the button says "Next", I don't see an extra space in the code either.

**Comment by alexsoble** [comment]
@Jacklynn I assigned this card back to you beacuse of the reproducibility question above, let me know if you still see something different — we can sync up to figure out why we may be seeing different things here.

**Comment by Jacklynn** [comment]
@alexsoble I just tested in safari and it's still happening for me. The easiest way to replicate for me:
* voice over on
* Go to second page
* get the focus on the "back button" 
* press the up arrow, to bring focus order to the Next button
* VO reads: "next button space" and shows a larger focus area than when tabbed.

**Comment by alexsoble** [comment]
Ooooh, thank you @Jacklynn! I can replicate the issue now and think I see a simple fix.

**Comment by Jacklynn** [comment]
FIXED thanks so much for your attention to detail.

Checked dev + safari + voice over
